hmy/downloader/downloader.go

```diff
@@ -203,6 +203,7 @@ func (d *Downloader) loop() {
 					time.Sleep(5 * time.Second)
 					trigger()
 				}()
+				time.Sleep(1 * time.Second)
```

Comment:
why is the 1s sleep added?

Reply:
1s sleep is to avoid infinite loop when having not enough peers. It is observed on testnet that the node 

1. Node start syncing process.
2. Node found it does not have enough peers and exit syncing with an error.
3. Node start another syncing because consensus / trigger 5s before.

This will cause an infinite loop that consumes CPU resource. Thus add a sleep mechanism here to avoid infinite loop.